gr-digital/python/digital/qa_mpsk_snr_est.py

```diff
@@ -19,40 +19,44 @@
 # the Free Software Foundation, Inc., 51 Franklin Street,
 # Boston, MA 02110-1301, USA.
 #
+""" Test digital.mpsk_snr_est_cc """
 
 import random
 
 from gnuradio import gr, gr_unittest, digital, blocks
 
+def random_bit():
+    """Create random bits using random() rather than randint(). The latter
+    changed for Python 3.2."""
+    return random.random() > .5
```

Comment:
My `import random` gives me a `random.randint()` on both Python2 and 3. Why the detour? You say "since 3.2, `randint` doesn't return the same integers. What do you mean with "not the same"?

Reply:
I've looked at the docs, and I see no trace of any change to randint. So, I really need clarification!

Reply:
Do you perhaps mean to do `version=1` when seeding, so that "older python number generatio compatibility" is restored? https://docs.python.org/3.6/library/random.html#random.seed

Reply:
No I mean the behavioural change in `randrange()`. See my comment below.

Reply:
Basically, we can't use `randint()` or `randrange()` if we need the same numbers across Py2 and 3.